### DISCUSSION TITLE
Are EJBCA 7.x and 8.x deployable on JBoss EAP 8?

### DISCUSSION THREAD

**Discussion by dangkhoaphung** [Keyfactor/ejbca-ce#593] [General]
Hello,

I'd like to understand whether EJBCA 7.x, 8.x can be deployed on JBoss EAP 8?

Thanks

**Comment by hesunmark** [comment]
Hi, 

Currently it is not possible to deploy EJBCA on Jakarta EE 10 Application servers such as JBoss EAP 8. However, EJBCA migration from JEE8 --> JEE10 is being worked on and should be completed later this year.

**Reply by dangkhoaphung** [reply]
Thank @hesunmark for your replying,
May I know whether it's EJBCA 9? And backward compatibility for EJBCA 7.x and 8.x is still there or no backward compatibility?

**Reply by primetomas** [reply]
It will be EJBCA 9.0 yes. 
EJBCA always have backwards compatibility on the database level. You have been able to upgrade EJBCA to new versions always. And so it will remain.
